virttest/utils_test/__init__.py

```diff
@@ -975,6 +975,9 @@ def env_check(self):
             logging.error("Unable to find python.")
             return False
         self.pip_bin = find_bin(self.session, ['pip', 'pip2', 'pip3'])
+        if not self.pip_bin:
+            logging.error("Unable to find pip.")
+            return False
```

Comment:
small nitpick, this check should be only performed when pip is required, which is only when `self.avocado_vt` is specified (line 981) or when `pip` is the `self.installtype` and Avocado is not yet installed (on line 1012). Otherwise we don't really need pip.

Reply:
@ldoktor  
Sorry I missed other cases
Yes, it needs to be fixed, but I feel that the error log should still exist if not found any pip binary path.
Or it doesn't have a sense?

Reply:
It'd be user-friendlier, but not really required...